src/info.plist

```diff
@@ -1117,29 +1350,56 @@
 			<key>type</key>
 			<string>alfred.workflow.output.notification</string>
 			<key>uid</key>
-			<string>970145EF-4ABC-4829-AA79-F48C6B355296</string>
+			<string>19D506AF-987F-4A62-A64F-34ACCF8501D0</string>
 			<key>version</key>
 			<integer>1</integer>
 		</dict>
 		<dict>
 			<key>config</key>
 			<dict>
-				<key>argument</key>
+				<key>concurrently</key>
+				<false/>
+				<key>escaping</key>
+				<integer>68</integer>
+				<key>script</key>
+				<string>function run() {
+  let query = "{query}";
+  let [, url] = query.split(",");
```

Comment:
The copy function currently copies an index instead of the URL. This can be resolved with the code below


```suggestion
  let [, , , url] = query.split(",");
```